test-suite/README.md

```diff
@@ -0,0 +1,130 @@
+# Getting started with Postman tests
+
+## Pre-reqs before running Postman tests
+1. Ensure you have built the docker containers by following the build process on the first README.md
+2. Postman installed
+3. Import the scripts into Postman `nia-patient-switching-standard-adaptor\test-suite\postman\Medicus Test Collection PSS Adaptor.postman_collection.json`
+4. Certificates have been added to postman
+    1. Open Postman / Settings (cog symbol)
+    2. Go to Certificates tab inside Settings menu
+    3. Turn on CA Certificates and add the `rootCA.pem` file from `nia-patient-switching-standard-adaptor\test-suite\postman\localhost-certificates`
+    4. Click on Add certificate next to Client Certificates
+    5. Add the `spineClient` file
+    6. Add the `spintClient.key`
+    7. Change the Host location to `localhost` : `443`
+    8. Click on Add
+    9. Close Settings menu
+
+## Setting up test-suite adaptors to run Postman tests
```

Comment:
The location needs to be added to `vars.sh` before the start script will run. Could we mention the `vars-versions.sh`  here too, as it's possible to use different versions of the docker images. Foe example if they need AMD64 images or they want to run older / newer images